assessment.sql

```diff
@@ -5,24 +5,48 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
+CREATE TABLE licensees(
+id SERIAL PRIMARY KEY,
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER VARYING,
+height INTEGER,
+weight INTEGER,
+dob DATE,
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+);
+
 -- your code here
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
+INSERT INTO licensees(surname, given_name, gender, height, weight, dob, eye_color, hair_color)
+VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
 
 -- your code here
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(surname, given_name, gender, height, weight, dob, eye_color, hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
+SELECT eye_color = 'Hazel' AND hair_color = 'Brown' OR hair_color = 'Black' FROM licensees
+SELECT * FROM licensees
+;
+
 
 -- your code here
 
 -- Question 5. Make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
 
--- your code here
+UPDATE licensees
+SET height = height - 1
+WHERE given_name = 'Marilynn Escobar' AND WHERE given_name = 'Chris Whaley'
```

Comment:
The first step to making this a working solution is to remove that second `WHERE` and leave everything else where it is. `WHERE` means that a condition follows; your entire condition for whether or not to select a row follows one `WHERE`.

Next, we have to decide just what it is that we're looking for. Are we looking for the person whose first name (`given_name`) is "Marilynn Escobar", or are we looking for the person whose first name is "Marilynn" and last name is "Escobar"?

So, then, we change `given_name = 'Marilynn Escobar'` to `given_name = 'Marilynn' AND surname = 'Escobar'`. Likewise for Chris Whaley.

Finally, we have to make one more decision regarding this condition. Are we trying to select the row representing the person whose name is both "Marilynn Escobar" **and** "Chris Whaley", or the rows of the people whose names are either "Marilynn Escobar" **or** "Chris Whaley"?

2/5